backend/game_handler.py

```diff
@@ -49,6 +106,15 @@ def reset_game(self):
             self.min_time, np.random.normal(self.average, self.std)
         )
 
+    def bid(self, ws, amount):
+        cash_available = self.game_handler_parent.cash[ws]
+        bid_amount = min(amount, cash_available)
```

Comment:
I like the idea, I'm debating on whether it's the way to go. We'll keep it for now :)

One thing that's for sure, is it needs to be sent back to the client. If client bids 5k but only has 1k, they need to get visual feedback that it only bid 1k.